apis/v1alpha1/gateway_types.go

```diff
@@ -132,11 +132,8 @@ type Listener struct {
 	//
 	// Support: Core
 	//
-	// +required
-	// +kubebuilder:validation:Minimum=0
-	// +kubebuilder:validation:Maximum=65536
-	// +kubebuilder:validation:ExclusiveMinimum=true
-	// +kubebuilder:validation:ExclusiveMaximum=true
+	// +kubebuilder:validation:Minimum=1
+	// +kubebuilder:validation:Maximum=65535
 	Port int32 `json:"port,omitempty"`
```

Comment:
Remove `omitempty` if this field is required and a default is not being set.

Reply:
Updated, thanks!